### ISSUE TITLE
[feature]: Offline mode, multiple workspace support in desktop app

### ISSUE THREAD

**Issue by baek-sang** [hoppscotch/hoppscotch#3514]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Summary

It would be nice to be able to use multiple workspaces without login.(offline mode)

### Why should this be worked on?

Depending on the service, may find it convenient to have multiple workspaces to work in.

**Comment by Igorgro** [comment]
Yes, this is №1 awaiting feature from hopscotch after Insomnia has introduced mandatory login.

**Comment by liyasthomas** [comment]
Reopening this issue for continued tracking as it was unintentionally closed. This is still relevant for further discussion/implementation. Thank you!

**Comment by SoCuul** [comment]
@liyasthomas If this is not present within the desktop app, would self-hosting an instance of hoppscotch be able to facilitate multiple workspaces without having to connect to external servers?

**Comment by liyasthomas** [comment]
@SoCuul Self-hosted Hoppscotch instances do not connect to external servers. Rest assured, multiple workspaces are available as well.

**Comment by SoCuul** [comment]
Nice to know! By chance, is there any way to use the desktop app but with the self-hosted server instead of the official one?

To be able to have multiple collections in the app, synced from the self-hosted server

**Comment by liyasthomas** [comment]
We're actively working on adding support for self-hosted instances in the desktop app, kindly subscribe to receive updates and track its progress on #3523.

**Comment by SoCuul** [comment]
Love how you guys are building Hoppscotch, especially after moving away from Insomnia ❤️
